### PR TITLE
Nicole Conway HW - number & API

### PR DESCRIPTION
**Pull request by nconway**
My homework for secret number game and API exercise

### CODE REVIEW

File: api01/github_api.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+git_info = JSON.load(RestClient.get('https://api.github.com/users/rails/repos'))
+
+#pulls html_url and watchers_count from API and puts them in arrays
+git_info.each do |pages|
+	watchers.push(pages['watchers_count'])
+	url.push(pages['html_url'])
+end 
+
+#creates a hash from the two arrays
+watchersurl = Hash[url.zip watchers]
```
Comment (dladowitz) [inline api01/github_api.rb]: Interesting. I didn't even know ruby had built in methods for merging two arrays into a hash

File: nicoleconway/number_game_nicoleconway.rb [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+		puts "Nope, too low.  You have " + (3 - num_guesses).to_s + " guesses left."
+		if num_guesses == 3
+			puts "You are out of guesses.  The secret number was " + secret_number.to_s
+		end
+	elsif guess > secret_number
+		num_guesses +=1
+		puts "Nope, too high.  You have " + (3 - num_guesses).to_s + " guesses left."
+		if num_guesses == 3
+			puts "You are out of guesses.  The secret number was " + secret_number.to_s
+		end
```
Comment (dladowitz) [inline nicoleconway/number_game_nicoleconway.rb]:
```ruby
if num_guesses == 3
    puts "You are out of guesses.  The secret number was " + secret_number.to_s
end
```
This logic shows up twice. Start trying to think of ways to not repeat the same blocks of code

**Comment by dladowitz** [comment]
Nice work. It all runs and I even learned a new method!
:+1:

**Comment by dladowitz** [comment]
I just noticed that your API homework is outside of your personal folder.
Can you move it into your folder and then do another Pull Request. 
thanks

**Comment by nconway** [comment]
Just put the API folder under my personal folder and created another pull
request.

Thanks,
Nicole

**Pull request merged**